tests/test_reports.py

```diff
@@ -51,3 +87,8 @@ def test_pdf_report_generation(capsys, nb, nb_config):
 
     # Check no stderr from vega CLI or similar
     assert capsys.readouterr().err == ""
+
+    # copy test report to $pwd so that it can be uploaded to github as an
+    # artifact
+    with open("test-report-generated.pdf", "wb") as handle:
```

Comment:
can you add `test-report-generated.pdf` to `.gitignore`? might want to also add `test-report-diff.pdf` in case someone runs the differ locally